test/visualTest.ts

```diff
@@ -174,12 +169,17 @@ describe("AsterPlot", () => {
                 expect(linesAfterResize.length).toBeLessThanOrEqual(numOfLabels);
                 expect(labelsAfterResize.length).toBeLessThanOrEqual(numOfLabels);
 
-                const firstLabelX: string = labels[0].getAttribute("x"),
-                    firstLabelY: string = labels[0].getAttribute("y"),
-                    lastLabelY: string = labels[labels.length - 1].getAttribute("y"),
-                    firstResizeLabelX: string = labelsAfterResize[0].getAttribute("x"),
-                    firstResizeLabelY: string = labelsAfterResize[0].getAttribute("y"),
-                    lastResizeLabelY: string = labelsAfterResize[labelsAfterResize.length - 1].getAttribute("y");
+                const firstLabel = labels[0],
+                    lastLabel = labels[labels.length - 1],
+                    firstResizeLabel = labelsAfterResize[0],
+                    lastResizeLabel = labelsAfterResize[labelsAfterResize.length - 1];;
```

Comment:
Please use "const" before each initialization, it makes code clearer.
Also, you can do it like this:
```js
const firstLabel = labels.at(0);
const lastLabel = labels.at(-1);

```

Reply:
Done.